fenix/app/src/main/java/org/mozilla/fenix/perf/StartupPathProvider.kt

```diff
@@ -100,7 +100,6 @@ class StartupPathProvider {
 
         override fun onStop(owner: LifecycleOwner) {
             // Clear existing state.
-            startupPathForActivity = StartupPath.NOT_SET
```

Comment:
This gets reset once the Activity is destroyed instead (overriding onDestroy here explicitly will mean this is called before the `HomeActivity` `onDestroy()` - I can't think of any particular bugs this may cause but welcome any thoughts.